deno_typescript/lib.deno_core.d.ts

```diff
@@ -1,5 +1,13 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
 
+// This file contains APIs that are introduced into the global namespace by
```

Comment:
I'd say that from the CLI user perspective we should place these APIS under Deno.internal.xyz rather than under Deno.core. The former suggests "these are built in functions", the latter tells the user "Unstable internal interface, do not use ".

Reply:
Let's discuss that in an issue, it's unrelated to this PR.